src/mgmt/tests/mocks.c

```diff
@@ -439,6 +439,13 @@ mock_ds_mgmt_exclude_setup(void)
 	uuid_clear(ds_mgmt_target_update_uuid);
 }
 
+void
+mock_ds_mgmt_drain_setup(void)
+{
+	ds_mgmt_target_update_return = 0;
+	uuid_clear(ds_mgmt_target_update_uuid);
+}
+
```

Comment:
It would be better to combine with the exclude setup function above, since they are the same. mock_ds_mgmt_target_update_setup or something like that.